src/server/game/Entities/Player/Player.cpp

```diff
@@ -17847,7 +17847,7 @@ void Player::_LoadAuras(PreparedQueryResult result, uint32 timediff)
             }
 
             // negative effects should continue counting down after logout
-            if (remaintime != -1 && !spellInfo->IsPositive())
+            if (remaintime != -1 && ((!spellInfo->IsPositive() && spellInfo->Id != 15007) || spellInfo->HasAttribute(SPELL_ATTR4_FADES_WHILE_LOGGED_OUT))) // Resurrection sickness should not fade while logged out
```

Comment:
Uh, I think it should fade when logged out but let me test it on retail first

Reply:
The issue isn't spellinfo from what I can see. It's a negative spell and the check for not positive traps it. The spell doesn't have the fades flag set and this is correct.

The question we need to ask is whether all the spells that have the flag are the correct list and if so, then we can drop the positive check. If not, then we keep in the exception.